packages/website/rulesMeta.ts

```diff
@@ -11,3 +11,7 @@ export const rulesMeta = Object.entries(rules).map(([name, content]) => ({
 }));
 
 export type RulesMeta = typeof rulesMeta;
+
+export type RuleMeta = RulesMeta[number];
+
+export type RuleMetaWithDocs = RuleMeta & { docs: object };
```

Comment:
Are these used anywhere? I couldn't find any usage for these and I can have a green (local) type-check without them.